lib/server.go

```diff
@@ -62,9 +62,9 @@ type Server struct {
 	eventManager  *EventManager
 	TxIndex       *TXIndex
 
-	handshakeController *HandshakeController
+	handshakeController *HandshakeManager
 	// fastHotStuffEventLoop consensus.FastHotStuffEventLoop
```

Comment:
Does server still need direct access to HandshakeManager? Ideally, NetworkManager can just intiantiate the HandshakeManager internally and route all messages to it. This way, all message handling for the handhsake process is encapsulated within NetworkManager